profiles/default/S3-OIDC-Testing.cyberduckprofile

```diff
@@ -20,36 +20,39 @@
         <key>Protocol</key>
         <string>s3</string>
         <key>Vendor</key>
-        <string>iterate GmbH</string>
+        <string>s3-sts</string>
         <key>Bundled</key>
         <true/>
+        <key>Description</key>
+        <string>S3 STS AssumeRoleWithWebIdentity</string>
         <key>Default Hostname</key>
-        <string>localhost:9000</string>
+        <string>localhost</string>
+        <key>Default Port</key>
+        <string>9000</string>
         <key>Default Nickname</key>
+        <string>MinIO</string>
         <key>Scheme</key>
         <string>http</string>
-        <string>MinIO</string>
         <key>Authorization</key>
-        <string>PasswordGrant</string>
```

Comment:
This is our testing profile. We tried to use the PasswordGrant flow to test the oAuth flow and the STS service seperately as discussed in the last meeting.